src/main/java/bio/terra/app/model/rawls/WorkspaceResponse.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Rawls
+ * Rawls API
+ *
+ * OpenAPI spec version: 1.0.0
+ *
+ *
+ * NOTE: This class is auto generated by the swagger code generator program.
+ * https://github.com/swagger-api/swagger-codegen.git
+ * Do not edit the class manually.
```

Comment:
Is this actually autogenned?  There are methods below that don't seem to be

Reply:
It started as autogenned, but then was augmented by me

Reply:
I'd like to know more about this.  Would this be overridden by future runs of the code generator?

Reply:
I've modified the comments. Should be clearer now!